### PR TITLE
Add method for Statistics access

### PR DESCRIPTION
**Pull request by nafur**
This PR adds a wrapper for `getStatistics()`.

### CODE REVIEW

File: cvc5_z3py_compat/z3.py [hunk 1]
```diff
@@ -5199,6 +5198,25 @@ def set(self, **kwargs):
...
+        >>> s.add(a == 0)
+        >>> s.check()
+        sat
+        >>> stats = s.statistics()
+        >>> stats['cvc5::CONSTANT']
+        {'defaulted': False, 'internal': False, 'value': {'integer type': 1}}
+        >>> len(stats.get()) < 10
+        True
+        >>> len(stats.get(True, True)) > 30
+        True
```
Comment (mpreiner) [inline cvc5_z3py_compat/z3.py:5216]: I'm not sure I understand what this should actually check?

Comment (mpreiner) [reply cvc5_z3py_compat/z3.py:5216]: Hmm, but this is only the docstring?

Comment (nafur) [reply cvc5_z3py_compat/z3.py:5216]: The docstrings are automatically analyzed for stuff that looks like a terminal session which is then used as unit tests (-> doctest)

Comment (mpreiner) [reply cvc5_z3py_compat/z3.py:5216]: Fancy.

**Review by mpreiner — APPROVED** [review]

**Pull request merged**